contribute/dev_review_process.rst

```diff
@@ -0,0 +1,41 @@
+.. OpenCMISS Review Process
+
+==============
+Review Process
+==============
+
+Check for the Green Tick
+========================
+
+Before accepting a tranche of work into the appropriate OpenCMISS prime repository, check that Jenkin's has tested and passed the code.  The status of the code is shown in the last commit of a pushed group of commits in the pull request.  The last commit will have a red cross for a failed build or a green tick for a passed build.  Make sure that the last commit has a green tick before merging.
+
+Check the Documentation
+=======================
+
+The documentation for the project is built as part of the testing process.  The details link at the bottom of the pull request web page will take you to the Jenkin's build of the library, this page shows the results of the unit tests for each target operating system, the results of the memory check test, and the results of the documentation build.
+
+.. The Documentation Builder link (entry 2. in step 5.) will take you to the build for the documentation.  On this page you can see the steps taken to build the documentation.  In the last step of the build (step 7.) there is a link 'dox' (entry 2.) that will take you to the built documentation.
+
+**Coming soon: how to check documentation using Jenkins**
```

Comment:
Stuck record. Future work belongs in separate issues, not littering our documentation.